### ISSUE TITLE
i3 workspace_auto_back_and_forth

### ISSUE THREAD

**Issue by ebetica** [nikitabobko/AeroSpace#43]
Would be great to have this feature!

https://i3wm.org/docs/userguide.html#workspace_auto_back_and_forth

**Comment by nikitabobko** [comment]
I agree that it would be nice to support

But I plan to support it as a flag for `workspace` command. Like: `workspace --auto-back-and-forth 1`. The downside will be that you need to mention the flag in all your `workspace` command usages in the config

I don't want to introduce a global `workspace_auto_back_and_forth` flag that changes the behavior of all `workspace` commands because I think such a global flag is error-prone:
1. It will make scripting for AeroSpace harder. Some scripts might expect that `workspace` command is idempotent
2. Some users might want to follow the focus when a window is moved to a different workspace only for specific workspaces:
    ```
    # 'workspace 1' is used in an idempotent manner in 'alt-1' binding. 
    # Global auto-back-and-forth is error-prone
    alt-1 = ['move-node-to-workspace 1', 'workspace 1']
    alt-2 = 'workspace --auto-back-and-forth 2'
    ```

**Comment by nikitabobko** [comment]
Released in 0.7.0-Beta
